### PR TITLE
Environment Forking

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 1]
```diff
@@ -71,14 +72,36 @@ If you are a visitor who does not belong to any public workspace, to send reques
...
+
+Your forked environment will be created in the selected workspace. You will be able to view the forked environments under the environment dropdown on the right side of Postman.
+
+<img alt="Environment dropdown for forked environments" src="https://assets.postman.com/postman-docs/environment-dropdown-view-v8.jpg" height="200px"/>
+
+Forks icon <img alt="Fork icon on right panel for v8" src="https://assets.postman.com/postman-docs/fork-icon-right-panel-v8.jpg" height="30px"/> on the right panel provides details about the forks created.
+
+<img alt="Fork icon overview for v8" src="https://assets.postman.com/postman-docs/fork-panel-overview-v8.jpg" height="300px"/>
+
+Click **View all forks** to reveal the list of fork labels and users who have active forks.
```
Comment (ankit-m) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:101]: This is not entirely correct. Replace with something like

`Click View all forks to reveal the detailed list of forks along with information about users who created them`

**Review by ankit-m — CHANGES_REQUESTED** [review]

**Review by ankit-m — APPROVED** [review]

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 2]
```diff
@@ -71,14 +72,36 @@ If you are a visitor who does not belong to any public workspace, to send reques
...
 Being a signed-in non-member, to send requests in a public workspace, fork the collection into a workspace that you belong to (either team or personal whichever you choose during fork creation) and then make changes.
 
-[![Visitor creating a fork](https://assets.postman.com/postman-docs/visitor-fork-collection-v2.jpg)](https://assets.postman.com/postman-docs/visitor-fork-collection-v2.jpg)
+<img src="https://assets.postman.com/postman-docs/visitor-fork-collection-v2.jpg" alt="Visitor creating a fork" height="400px"/>
 
 > Make sure your public profile is enabled before you fork a collection from a public workspace.
 
+## Forking an environment
+
+To fork an environment in Postman, select the environment in the __Environments__ sidebar, click __View more actions__ (__...__), and select __Create a fork__. You can also fork an environment by clicking __Fork__ button in environment overview tab.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:85]: Let's remove "button"

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 3]
```diff
@@ -71,14 +72,36 @@ If you are a visitor who does not belong to any public workspace, to send reques
...
+
+Enter a label for your fork, and select a workspace to save it to. Click __Fork Environment__.
+
+<img src="https://assets.postman.com/postman-docs/fork-environment-v8.jpg" alt="Fork Label Environment" height="400px"/>
+
+Your forked environment will be created in the selected workspace. You will be able to view the forked environments in the sidebar on the left as well as under the environment dropdown on the right side of Postman.
+
+<img alt="Environment dropdown for forked environments" src="https://assets.postman.com/postman-docs/environment-dropdown-view-v8.jpg" height="200px"/>
+
+Forks icon <img alt="Fork icon on context bar for v8" src="https://assets.postman.com/postman-docs/fork-icon-right-panel-v8.jpg" height="30px"/> on the context bar provides details about the forks created.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:97]: Let's add a "The" at the front

**Review by claireriley — APPROVED** [review]
Looks great! Two very tiny edits

**Pull request merged**